package.json

```diff
@@ -78,15 +78,19 @@
             "@babel/preset-env"
         ],
         "plugins": [
-          [
-            "@babel/plugin-transform-runtime", {
-            "corejs": false,
-            "regenerator": true
-          }
-          ]
+            [
+                "@babel/plugin-transform-runtime",
+                {
+                    "corejs": false,
+                    "regenerator": true
+                }
+            ]
         ]
```

Comment:
Might cause merge conflict

Reply:
As long as there are no potential merge conflicts reported by github we can leave this :)